vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/video/ScreenRecorderTestRunListener.kt

```diff
@@ -45,11 +47,14 @@ class ScreenRecorderTestRunListener(
 
     private var hasFailed: Boolean = false
```

Comment:
Is it okay that we store `hasFailed` status in each listener separatly?

Reply:
Each listener is created per batch. If you want I can rename the variable to hasBatchFailed

Reply:
It will be clearer or could rename `TestRunListener` to `TestBatchListener`

Reply:
Done